src/core/client/admin/local/initLocalState.ts

```diff
@@ -17,6 +17,7 @@ const initLocalState: InitLocalState = async ({
   environment,
   context,
   auth = null,
+  staticConfig,
```

Comment:
This `staticConfig` _must_ be passed to the later call to `initLocalBaseState`. TS is happy now because the type of the parameter is `StaticConfig | null | undefined`.

Reply:
No problem, is fixed, just updating some tests and this will be pushed up.